samples/ui-library/src/components/UILibraryComponent/Buttons.tsx

```diff
@@ -0,0 +1,56 @@
+import Button from "@geocortex/web/ui/button";
+import React from "react";
+import UILibraryComponentModel from "./UILibraryComponentModel";
+
+interface ButtonsProps {
+    model: UILibraryComponentModel;
+}
+
+export default function Buttons({ model }: ButtonsProps) {
+    const handleClick = (
+        event: React.MouseEvent<HTMLButtonElement, MouseEvent>
+    ) => {
+        model.messages.commands.ui.alert.execute({
```

Comment:
Minor thing: although an object's injected dependencies are public properties, I consider it bad form for code outside of that object to rely on and directly reference its dependencies. In this case the pattern I would promote would be using UIContext:
```
const { commands } = useContext(UIContext);
const handleClick = (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>
 ) => {
     commands.ui.alert.execute({
          message: "Button clicked!",
     });
};
```

Reply:
Ah of course. Much better pattern. Removed the prop.